aws-spi-pekko-http/src/test/scala/org/apache/pekko/stream/connectors/awsspi/PekkoHttpClientSpec.scala

```diff
@@ -17,14 +17,21 @@
 
 package org.apache.pekko.stream.connectors.awsspi
 
-import java.util.Collections
+import com.typesafe.config.ConfigFactory
 
+import java.util.Collections
 import org.apache.pekko
+import org.apache.pekko.http.scaladsl.settings.{ ClientConnectionSettings, ConnectionPoolSettings }
```

Comment:
can you remove the 'org.apache' part of this import as `import org.apache.pekko` is already in scope?